main.nf

```diff
@@ -791,7 +870,7 @@ process dada_trunc_parameter {
 	!params.untilQ2import
 
 	script:
-	if( !params.trunclenf || !params.trunclenr ){
+	if( params.trunclenf == false || params.trunclenr == false ){
```

Comment:
Why this change? This seems inconsistent with the rest of the pipeline.

Reply:
This change is made to allow for trunclenr and trunclenf to be set to the value 0, which is interpreted in DADA2 as "no trunctaion". Needed for ITS sequences where we don't expect a given length.

Reply:
Right, makes sense!